doc/changelog/10-standard-library/12044-issue-12015.rst

```diff
@@ -0,0 +1,9 @@
+- **Fixed:**
+  Rewritten Structures.OrderedTypeEx.String_as_OT.compare
+  to avoid huge proof terms.
+  (`#12044 <https://github.com/coq/coq/pull/12044>`,
+   by formalize.eth (formalize@protonmail.com)).
+- **Added:**
+  Added Structures.OrderedTypeEx.Ascii_as_OT.
+  (`#12044 <https://github.com/coq/coq/pull/12044>`,
+   by formalize.eth (formalize@protonmail.com)).
```

Comment:
```suggestion
- **Fixed:**
  Rewrote ``Structures.OrderedTypeEx.String_as_OT.compare``
  to avoid huge proof terms
  (Fixes `#12015 <https://github.com/coq/coq/issues/12015>`_,
  `#12044 <https://github.com/coq/coq/pull/12044>`_,
   by formalize.eth (formalize@protonmail.com)).
- **Added:**
  Added ``Structures.OrderedTypeEx.Ascii_as_OT``
  (`#12044 <https://github.com/coq/coq/pull/12044>`_,
   by formalize.eth (formalize@protonmail.com)).
```
The underscores are important for rendering, and the periods should only occur at the very end of the sentence, not before the parenthetical.